### DISCUSSION TITLE
Is it possible to have multiple collections_dir?

### DISCUSSION THREAD

**Discussion by evigneshkumar** [jekyll/jekyll#9328] [General]
I have three directories ps, mee, and foo. 
Each directory consists of  multiple collections.

Now, I updated config.yml as:
 

collections_dir: ps, mee, foo 
collections:
  "2018.3.1":
    version: "2018.3.1"
    output: true
    permalink: /ps/:collection/:categories/:slug/
  "mee_0.0.1":
    version: "0.0.1"
    output: true
    permalink: /mee/:collection/:categories/:slug/
    product: MEE
    order: 1
  "foo_.0.1":
    version: "0.0.1"
    output: true
    permalink: /foo/:collection/:categories/:slug/
    product: MEE
    order: 1
    
But _site is not being rendered. 

Structure

├── assets
│   └── css
│       └── main.scss
├── ps  # This is a collections directory
│   ├── _2018.3.1  #this is collection
│   │     │── about  # this is categories
│   │   │   ├── doc1.md          
│   │   │   ├── doc2.md           
│   │   │  └── doc3.md         
├── mee  # This is a collections directory
│   ├── mee_0.0.1  #this is collection
│    │     │── about  # this is categories
│   │   │   ├── doc1.md           
│   │   │   ├── doc2.md           
│   │   │   └── doc3.md           
├── foo  # This is a collections directory 
│   ├── _foo_0.0.1  #this is collection
│   │     │── about  # this is categories
│       ├── doc1.md
│       ├── doc2.md
│       └── doc3.md
├── _config.yml
├── _data
├── _includes
├── _layouts
│   └── default.html
└── _sass
    └── main.scss

**Comment by ashmaroli** [comment]
Hello @evigneshkumar.
Jekyll doesn't support multiple *collection_dir*. It is designed to be a "basket" that contains all collections (including posts) in the site.
